### PR TITLE
support VM compute and controller nodes

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

**Review by jackhodgkiss — APPROVED** [review]
LGTM.

File: compute_instances.tf [hunk 1]
```diff
@@ -60,6 +60,17 @@ resource "openstack_compute_instance_v2" "compute" {
   network {
     name = var.multinode_vm_network
   }
+  dynamic "block_device" {
+    for_each = var.compute_is_vm ? [1] : []
+    content {
+      uuid                  = data.openstack_images_image_v2.multinode_image.id
+      source_type           = "image"
+      volume_size           = 100
```
Comment (markgoddard) [inline compute_instances.tf:68]: An alternative would be to make volume size configurable, and allow 0 = no volume.

Comment (Alex-Welsh) [reply compute_instances.tf:68]: Would it also be useful to set the seed/storage/ansible control size in the same way?

Comment (markgoddard) [reply compute_instances.tf:68]: Could be. I don't know how much of that 100G we actually use.

**Pull request merged**